nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_width.py

```diff
@@ -204,19 +217,25 @@ def get_state(self) -> Dict[str, Any]:
             self._state_names.MAX_NUM_WIDTHS: self.max_num_widths,
             self._state_names.WIDTH_STEP: self.width_step,
             self._state_names.WIDTH_MULTIPLIERS: self.width_multipliers,
-            self._state_names.FILTER_IMPORTANCE: self.filter_importance
+            self._state_names.FILTER_IMPORTANCE: self.filter_importance,
+            self._state_names.OVERWRITE_GROUPS: self.overwrite_groups,
+            self._state_names.OVERWRITE_GROUPS_WIDTHS: self.overwrite_groups_widths,
+            self._state_names.ADD_DYNAMIC_INPUTS: self.add_dynamic_inputs
         }
 
     def __eq__(self, other: 'ElasticWidthParams') -> bool:
         return self.__dict__ == other.__dict__
 
+    def __str__(self):
+        return f"{self.__class__.__name__}: width_step: {self.width_step} min_width: {self.min_width} width_multipliers: {self.width_multipliers} max_num_widths: {self.max_num_widths} overwrite_groups: {self.overwrite_groups} overwrite_group_widths: {self.overwrite_groups_widths}"
+
 
 class ElasticOutputWidthOp(ElasticWidthOp):
     """
     Base class for trimming output channels (output width) of the operations.
     """
 
-    def __init__(self, max_width: int, node_name: str, params: ElasticWidthParams):
+    def __init__(self, max_width: int, node_name: str, params: ElasticWidthParams, fixed_width_list=[]):
```

Comment:
Would you please add a type hint for consistency?

Reply:
Added Type Hint. Thanks!